src/pages/index.tsx

```diff
@@ -148,13 +174,26 @@ export default function Home() {
               <p
                 key={item}
                 onClick={() => changeTopic(item)}
-                className={`bg-gray-300 ${textColor} px-2 py-1 rounded mr-2 mb-2 cursor-pointer`}
+                className={cn(
+                  `bg-gray-300 px-2 py-1 rounded mr-2 mb-2 cursor-pointer`,
+                  textColor,
+                  selectedLabels.includes(item) && "bg-blue-500 text-white"
+                )}
                 style={{ backgroundColor: `#${item}` }}
               >
                 {item}
               </p>
             );
           })}
+          {!!selectedLabels.length && (
```

Comment:
Put the button in the tag section and make it hide when there is no selected tag